apertium_apy/apy.py

```diff
@@ -19,6 +19,7 @@
 import sys
 from datetime import timedelta
 from logging import handlers as logging_handlers  # type: ignore
+import pkg_resources
```

Comment:
We want to use the default `langNames.db` if the `pkg_resources` module isn't installed (and catch the error).

Reply:
Okay I will add that condition